### PR TITLE
Fix image

### PR DESCRIPTION
**Pull request by mblue9**

### CODE REVIEW

**Comment by mblue9** [comment]
Getting this error in the actions
```
Quitting from lines 154-155 (introduction.Rmd) 
Error: processing vignette 'introduction.Rmd' failed with diagnostics:
tidybulk says: using SummarizedExperiment with aggregate_duplicates, you need to specify .transcript parameter. It should be a feature-wise column (e.g. gene symbol) that you want to collapse he features with (e.g. ensembl). It cannot be the representation of rownames(SummarizedExperiment), as those are unique by definition, and not part of rowData per-se.
```

File: vignettes/introduction.Rmd [hunk 1]
```diff
@@ -149,8 +151,7 @@ tidybulk provide the `aggregate_duplicates` function to aggregate duplicated tra
 <div class="column-left">
 TidyTranscriptomics
 ```{r aggregate, message=FALSE, warning=FALSE, results='hide', class.source='yellow'}
-rowData(counts_SE)$gene_name = rownames(counts_SE)
-counts_SE.aggr = counts_SE %>% aggregate_duplicates(.transcript = gene_name)
+counts_SE.aggr = counts_SE %>% aggregate_duplicates()
```
[the remark this answers is not in the document]
Comment (stemangiola) [reply vignettes/introduction.Rmd:154]: @mblue9

**Comment by stemangiola** [comment]
@mblue9 hello

**Review by stemangiola — CHANGES_REQUESTED** [review]

**Comment by mblue9** [comment]
Superceded by https://github.com/stemangiola/tidybulk/pull/254

**Pull request closed without merging**